### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by Andersama**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -13351,6 +13351,16 @@
         <Type>Script</Type>
         <Description>Happenlance Developer AutoSplitter</Description>
         <Website>https://github.com/pmttavara/HappenlanceAutoSplitter</Website>
+    </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Inscryption</Game>
+        </Games>
+        <URLs>
+            <URL>https://gist.github.com/Andersama/2160d1cfc1c95341607b728ba39d538f</URL>
```
Comment (NoTeefy) [inline LiveSplit.AutoSplitters.xml:13360]: @Andersama You need to provide the RAW url to a file on a GitHub or Gitlab repository.
Gists aren't supported. Just change it in a second commit and I'll be happy to merge your request :)

**Review by NoTeefy — CHANGES_REQUESTED** [review]

File: LiveSplit.AutoSplitters.xml (hunk 1 above)
Comment (Andersama) [reply LiveSplit.AutoSplitters.xml:13360]: ah, that would make sense

File: LiveSplit.AutoSplitters.xml [hunk 2]
```diff
@@ -13351,6 +13351,16 @@
...
         <Description>Happenlance Developer AutoSplitter</Description>
         <Website>https://github.com/pmttavara/HappenlanceAutoSplitter</Website>
+    </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Inscryption</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/Andersama/LiveSplitters/main/Inscryption.asl</URL>
+        </URLS>
```
Comment (NoTeefy) [inline LiveSplit.AutoSplitters.xml:13361]: and now there's a case-sensitive problem here. The tags are matched with case-sensitiveness. You'd need to change the </URLS> to </URLs>

Comment (NoTeefy) [inline LiveSplit.AutoSplitters.xml:13361]: and now there's a case-sensitive problem here. The tags are matched with case-sensitiveness. You'd need to change the to

**Review by NoTeefy — CHANGES_REQUESTED** [review]

**Pull request merged**